test_scripts/Polices/External_UCS/003_ATF_P_TC_LPT_Creation_PreloadedPT_without_disallowed_by_external_consent_entities_off_struct.lua

```diff
@@ -0,0 +1,137 @@
+-- 2. Check SDL status
+-- 3. Register app
+-- 4. Activate app
+-- 5. Verify PTSnapshot
+--
+-- Expected result:
+-- a. Status = 1 (SDL is running)
+-- b. PTSnapshot doesn't contain 'disallowed_by_external_consent_entities_off' section
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+  config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+  config.defaultProtocolVersion = 2
+
+--[[ Required Shared Libraries ]]
+  local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+  local commonSteps = require('user_modules/shared_testcases/commonSteps')
+  local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+  local sdl = require('SDL')
+  local testCasesForExternalUCS = require('user_modules/shared_testcases/testCasesForExternalUCS')
+
+--[[ Local variables ]]
+  local grpId = "Location-1"
+  local checkedSection = "disallowed_by_external_consent_entities_off"
+
+--[[ General Precondition before ATF start ]]
+  commonFunctions:SDLForceStop()
+  commonSteps:DeleteLogsFileAndPolicyTable()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+  testCasesForExternalUCS.removePTS()
+
+--[[ General Settings for configuration ]]
+  Test = require("user_modules/connecttest_resumption")
+  require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+  commonFunctions:newTestCasesGroup("Preconditions")
+
+  function Test:CheckSDLStatus()
+    testCasesForExternalUCS.checkSDLStatus(self, sdl.RUNNING)
+  end
+
+  function Test:StopSDL()
+    testCasesForExternalUCS.ignitionOff(self)
+  end
+
+  function Test:CheckSDLStatus()
+    testCasesForExternalUCS.checkSDLStatus(self, sdl.STOPPED)
+  end
+
+  function Test.RemoveLPT()
+    testCasesForExternalUCS.removeLPT()
+  end
+
+  function Test.UpdatePreloadedPT()
+    local updateFunc = function(preloadedTable)
+      preloadedTable.policy_table.functional_groupings[grpId][checkedSection] = nil
+    end
+    testCasesForExternalUCS.updatePreloadedPT(updateFunc)
+  end
+
+--[[ Test ]]
+  commonFunctions:newTestCasesGroup("Test")
+
+  function Test.StartSDL()
+    StartSDL(config.pathToSDL, config.ExitOnCrash)
+    os.execute("sleep 1")
+  end
+
+  function Test:CheckSDLStatus()
+    testCasesForExternalUCS.checkSDLStatus(self, sdl.RUNNING)
+  end
+
+  function Test:InitHMI()
+    self:initHMI()
+  end
+
+  function Test:InitHMI_onReady()
+    self:initHMI_onReady()
+  end
+
+  function Test:ConnectMobile()
+    self:connectMobile()
+  end
+
+  function Test:StartSession()
+    testCasesForExternalUCS.startSession(self, 1)
+  end
+
+  function Test:RAI()
+    testCasesForExternalUCS.registerApp(self, 1)
+  end
+
+  function Test:ActivateApp()
+    testCasesForExternalUCS.activateApp(self, 1)
+  end
+
+  function Test:CheckPTS()
+    if testCasesForExternalUCS.pts.policy_table.functional_groupings[grpId][checkedSection] ~= nil then
```

Comment:
@dboltovskyi,
1. testCasesForExternalUCS.pts is not assigned in script.
2. In case sdl_snapshot.json is not created you will have runtime error. The same if the other elements iare not created.